internal/pkg/term/selector/selector.go

```diff
@@ -1022,38 +1023,80 @@ func filterDeployedServices(filter DeployedServiceFilter, inServices []*Deployed
 	return outServices, nil
 }
 
-// Topics asks the user to select from all Copilot-managed SNS topics and returns the ARNs
-// of those topics.
-func (s *DeploySelect) Topics(promptMsg, help, app, env string) ([]string, error) {
-	topics, err := s.deployStoreSvc.ListSNSTopics(app, env)
+// Topics asks the user to select from all Copilot-managed SNS topics *which are deployed
+// across all environments* and returns the topic structs.
```

Comment:
nicee! I was wondering how we were going to get to have a manifest that's guaranteed to deploy by default 👏